src/components/Fields/TextArea.story.js

```diff
@@ -40,6 +40,28 @@ stories.add(
             placeholder="your placeholder here"
             value="Hello, Friend"
         />,
+        <div>
+            <TextArea
+                label="Number Zero"
+                placeholder="your placeholder here"
+                value={0}
```

Comment:
`TextArea` doesn't accept numbers, so this logs an error to the console.

Reply:
valid point, I'll remove!